examples/tests/client-tests.js

```diff
@@ -0,0 +1,182 @@
+/* global QUnit */
+
+'use strict';
+
+const BASE_MAIN_CHILDREN_NUMBER = 3;
+
+QUnit.module('Pictures');
+
+QUnit.test(
+  'putPicturesInPage',
+  (assert) => {
+    assert.ok(
+      typeof window.forTesting.putPicturesInPage === 'function',
+      'Create a function `putPicturesInPage` receiving an array of pictures. The function will add each picture and its attributes to a different section with the class `picture`.',
+    );
+
+    const elM = document.querySelector('main');
+    assert.strictEqual(
+      elM.childElementCount,
+      (0+BASE_MAIN_CHILDREN_NUMBER),
+      'Before using the function, the element `main` must be empty of pictures.',
```

Comment:
you could also do an assertion with `querySelector('main .picture') == null` rather than looking only by number or children of `main`.